src/Akka.HealthCheck.Persistence/AkkaPersistenceLivenessProbe.cs

```diff
@@ -166,19 +177,22 @@ protected override bool Receive(object message)
 
         protected override void PreStart()
         {
-            CreateProbe(true);
+            CreateProbe();
         }
 
-        private void CreateProbe(bool firstTime)
+        private void CreateProbe()
         {
-            _probe = Context.ActorOf(Props.Create(() => new SuicideProbe(Self, firstTime, _id)));
-            if(firstTime)
+            var isEven = _probeCounter % 2 == 0;
+            _probeCounter++;
+            _probe = Context.ActorOf(Props.Create(() => new SuicideProbe(Self, isEven, _id)));
+            
+            if(isEven)
             {
-                _probe.Tell("hit" + _probeCounter++);
+                _probe.Tell("hit");
             }
             else
             {
-                Context.System.Scheduler.ScheduleTellOnce(_delay, _probe, "hit" + _probeCounter++, Self);
```

Comment:
why don't we version this any longer?

Reply:
[Chesterton's Fence](https://fs.blog/chestertons-fence/) - why remove it if it's not hurting anything? I can't tell you, given how old the code is, why it was there in the first place.